geopandas/base.py

```diff
@@ -142,7 +142,7 @@ def crs(self):
         Examples
         --------
 
-        >>> s.crs
+        >>> s.crs  # doctest: +SKIP
```

Comment:
All outputs of `crs` are skipped because `__repr__` returns a blank line below the text which is seen as trailing whitespace in docstring and hence removed.